app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<h1><%= @application.name %></h1>
+
+<h3>Street Address: <%= @application.street_address %></h3>
+<h3>City: <%= @application.city %></h3>
+<h3>State: <%= @application.state %></h3>
+<h3>Zip Code: <%= @application.zip_code %></h3>
+<h3>Reason for applying: <%= @application.description %></h3>
+<h3>Application Status: <%= @application.status %></h3>
+
+<h3>Applying for pets: </h3>
+<ul>
+    <% @application.pets.each do |pet| %>
+        <li><p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></p></li>
+        <% if ApplicationPet.where({pet_id: pet.id,application_id: @application.id}).first.status == "Approved" %>
+            <%= "(#{pet.name} already approved)" %>
+        <% elsif ApplicationPet.where({pet_id: pet.id,application_id: @application.id}).first.status == "Rejected" %>
+            <%= "(#{pet.name} already rejected)" %>
+        <% else %>
+            <%= form_with url: "/admin/applications/#{@application.id}", method: :get, local: true do |f| %>
+                <%= f.hidden_field :approved_pet_id, value: pet.id %>
+                <%= f.submit "Approve #{pet.name}" %>
+            <% end %>
+            <%= form_with url: "/admin/applications/#{@application.id}", method: :get, local: true do |f| %>
+                <%= f.hidden_field :rejected_pet_id, value: pet.id %>
+                <%= f.submit "Reject #{pet.name}" %>
+            <% end %>
+        <% end %>   
+    <% end %>
+</ul>
+
+<%# <p><%= link_to "Update #{@application.name}", "pets/#{@application.id}/edit" %></p>
+<%# #<p><%= link_to "Delete #{@application.name}", "/pets/#{@application.id}", method: :delete %></p>
+
+<% if @application.status == "In Progress" %>
```

Comment:
For the admin page, you don't need to have the search functionality included, just FYI! That's just for the applicant to add to their application before submitting